spec/controllers/cookies_consent_controller_spec.rb

```diff
@@ -1,35 +1,39 @@
 require 'spec_helper'
 
 RSpec.describe CookiesConsentController, type: :controller do
-  describe 'POST accept_cookies' do
-    subject(:response) { post :accept_cookies }
+  describe 'POST create' do
+    subject(:response) { post :create, params: { cookie_acceptance: acceptance } }
 
-    let(:expected_cookies) do
-      '{"settings":true,"usage":"true","remember_settings":"true"}'
-    end
+    describe 'with accept' do
+      let(:acceptance) { 'accept' }
 
-    it 'sets the cookie policy correctly' do
-      expect(response.cookies['cookies_policy']).to eq(expected_cookies)
-    end
+      let(:expected_cookies) do
+        '{"settings":true,"usage":"true","remember_settings":"true"}'
```

Comment:
Do you mind adding some spaces between the key/value pairs?

Reply:
Yeh good shout - I was trying to minimize the LOC i was changing to keep the diff simple